target/linux/sunxi/config-5.10

```diff
@@ -376,7 +380,6 @@ CONFIG_RESET_SUNXI=y
 CONFIG_RFS_ACCEL=y
 CONFIG_RPS=y
 CONFIG_RWSEM_SPIN_ON_OWNER=y
-CONFIG_SATA_PMP=y
```

Comment:
Note that this is also a functional regression. #4292
But I don't think we need to take care of it right now...(until someone yells at it)